README.md

```diff
@@ -5,10 +5,8 @@
+|**master**|[![Build Status](https://dnceng.visualstudio.com/public/_apis/build/status/dotnet/roslyn/roslyn-CI?branchName=master)](https://dnceng.visualstudio.com/public/_apis/build/status/dotnet/roslyn/roslyn-CI?branchName=master)|[![Build Status](https://dnceng.visualstudio.com/public/_apis/build/status/dotnet/roslyn/roslyn-CI?branchName=master)](https://dnceng.visualstudio.com/public/_apis/build/status/dotnet/roslyn/roslyn-CI?branchName=master)|[![Build Status](https://dnceng.visualstudio.com/public/_apis/build/status/dotnet/roslyn/roslyn-CI?branchName=master)](https://dnceng.visualstudio.com/public/_apis/build/status/dotnet/roslyn/roslyn-CI?branchName=master)|[![Build Status](https://dnceng.visualstudio.com/public/_apis/build/status/dotnet/roslyn/roslyn-CI?branchName=master)](https://dnceng.visualstudio.com/public/_apis/build/status/dotnet/roslyn/roslyn-CI?branchName=master)|[![Build Status](https://dnceng.visualstudio.com/public/_apis/build/status/dotnet/roslyn/roslyn-CI?branchName=master)](https://dnceng.visualstudio.com/public/_apis/build/status/dotnet/roslyn/roslyn-CI?branchName=master)|[![Build Status](https://ci.dot.net/buildStatus/icon?job=dotnet_roslyn/master/windows_debug_vs-integration)](https://ci.dot.net/job/dotnet_roslyn/job/master/job/windows_debug_vs-integration/)|[![Build Status](https://ci.dot.net/buildStatus/icon?job=dotnet_roslyn/master/windows_release_vs-integration)](https://ci.dot.net/job/dotnet_roslyn/job/master/job/windows_release_vs-integration/)|[![Build Status](https://ci.dot.net/buildStatus/icon?job=dotnet_roslyn/master/windows_debug_spanish_unit32_prtest)](https://ci.dot.net/job/dotnet_roslyn/job/master/job/windows_debug_spanish_unit32_prtest/)
```

Comment:
The links on the badges seem to just take you to the badge rather than to the failed build.  The build link should be something like https://dev.azure.com/dnceng/public/_build/index?definitionId=15&branchName=master

Reply:
Sigh ... the suggested badges really should work here vs. trying to get us to hand generate them 😦